content-cache_rock/rockcraft.yaml

```diff
@@ -15,6 +15,8 @@ parts:
       - nginx-light
       - bash
       - coreutils
+    stage-snaps:
+      - gtrkiller-nginx-prometheus-exporter/latest/edge
```

Comment:
Should this include your handle? Or should it be more general?

Reply:
given that there is already an NGINX Prometheus Exporter without the handle, but that we cannot edit at  all https://snapcraft.io/nginx-prometheus-exporter I think leaving my handle there for now is useful to not confuse my snap with the other one.